tests/libs/benchmark/include/benchmark/MemoryPoller.h

```diff
@@ -70,8 +70,8 @@ class MemoryPoller
   std::mutex _mutex_started;
   std::condition_variable _cond_var_started;
 
-  bool _term;
   bool _run;
+  bool _term;
```

Comment:
Does strict build also catch the order of variable ..?

Reply:
Right. Constructor's initialization order should be same with order of variable.